aider.el

```diff
@@ -22,6 +22,11 @@
   :prefix "aider-"
   :group 'convenience)
 
+(defvar aider-read-string-history nil
+  "History list for aider read string inputs.")
+(with-eval-after-load 'savehist
```

Comment:
Thanks for the work! I suspect this PR is to keep a dedicate history for aider-plain-read-string, and maybe it can persist after emacs close? so next time emacs can still have the history.

I am not familiar with 'savelist variable actually. Just want to confirm it can be loaded automatically? If the PR has been tested, I'll just go ahead and merge it.